rental.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'book'
+require_relative 'person'
+
+attr_accessor :date, :book, :person
+
+class Rental
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
+
+  def add_rental
+    book.rentals << self
+  end
+
+  def add_to_person
+    person.rentals << self
+  end
```

Comment:
- [ ]  The rental should be adding itself to the instance variables @book and @person achieving the 'belongs-to' association. Rather than creating methods for adding rentals to the book and person, try to implement that in the constructor.

```ruby
def initialize(date, book, person)
    .
    .

    @book = book
    book.rentals << self

    @person = person
    person.rentals << self
 end
```